packages/dev/core/src/Materials/PBR/pbrBaseMaterial.ts

```diff
@@ -1714,6 +1714,7 @@ export abstract class PBRBaseMaterial extends PushMaterial {
                         if (reflectionTexture.irradianceTexture) {
                             defines.USEIRRADIANCEMAP = true;
                             defines.USESPHERICALFROMREFLECTIONMAP = false;
+                            defines.USESPHERICALINVERTEX = false;

```

Comment:
This fixed a shader compile error when enabling the irradianceTexture on the fly.